app/models/concerns/track_activity.rb

```diff
@@ -218,6 +218,14 @@ def transfer_activity_parameters(params, activity) # rubocop:disable Metrics/Abc
                     })
     end
 
+    if activity.parameters[:action] == 'group_sample_clone'
+      params.merge!({
+                      target_project_puid: activity.parameters[:target_project_puid],
```

Comment:
We should only need to merge the cloned_samples_count since the activity listing at the group level just displays that samples were copied to other projects with the actual details being contained within the more details modal